addons/website_slides_survey/models/slide_slide.py

```diff
@@ -119,8 +119,8 @@ def _generate_certification_url(self):
             if slide.channel_id.is_member:
                 user_membership_id_sudo = slide.user_membership_id.sudo()
                 # Users can Rejoin the course and he/she may have the previous Inputs
-                # Don't consider the Completed User input and Pick the only uncompleted one.
-                if user_membership_id_sudo.user_input_ids.filtered(lambda user_input: user_input.state != 'done'):
+                # The relationship between the membership and previous user_input is removed.
```

Comment:
I would actually remove both comment lines, they don't make much sense anymore, we just consider existing inputs from the membership now.